docs/src/roots.md

```diff
@@ -366,15 +366,15 @@ dfᵏs (generic function with 1 method)
 
 ```jldoctest roots
 julia> Roots.newton(f, D(f), 2)
-2.0945514815423265
+3.141592653589793
 
 ```
 
 Or, for Halley's method:
 
 ```jldoctest roots
 julia> Roots.halley(f, D(f), D(f,2), 2)
-2.0945514815423265
+3.141592653589793
```

Comment:
This must  be the same issue. I wonder if D or f is not matching the example.

Reply:
Let me chack again.